crates/uplc/Cargo.toml

```diff
@@ -16,7 +16,7 @@ exclude = ["test_data/*"]
 cryptoxide = "0.4.2"
 flat-rs = { path = "../flat", version = "0.0.7" }
 hex = "0.4.3"
-minicbor = { version = "0.18.0", features = ["std"] }
```

Comment:
This can be changed back. Was a mistake on my part with trying the wrong way to pull out an i64 from PlutusData's BigInt enum

Reply:
so change it back then?

Reply:
done